app/ui/src/assets/dictionary/en-GB.json

```diff
@@ -20,11 +48,59 @@
       "customizations": "Customizations",
       "settings": "Settings"
     },
+    "header": {
+      "navbartoggle": "Toggle navigation",
+      "helpmenu": {
+        "sampletutorials": "Sample {{integrations.integration}} Tutorials",
+        "userguide": "User Guide",
+        "support": "Support",
+        "contactus": "Contact Us"
+      },
+      "usermenu": {
+        "logout": "Logout"
+      }
+    },
+    "settings": {
+      "oauth-apps": {
+        "heading": "OAuth Application Management",
+        "description": "To connect to an application that uses the OAuth protocol, obtain a client ID and a client secret from the application. See the <a href=\"https://access.redhat.com/documentation/en-us/red_hat_jboss_fuse/7.0-tp/html-single/integrating_applications_with_ignite/#obtaining-authorization-to-access-applications\" target=\"_blank\" rel=\"nofollow\">documentation</a> for help.",
+        "registration-instruction": "During registration, enter this callback URL: {{0}}",
+        "list": {
+          "item": {
+            "description": "Client ID and client secret not configured.",
+            "remove-btn-modal": "Are you sure you want to clear the credentials for '{{0}}'?"
+          }
+        },
+        "form": {
+          "success": {
+            "msg": "Registration successful!",
+            "action": {
+              "msg": "You can validate these values when {{0}} to {{1}}.",
+              "link": "<a href=\"{{0}}\">creating a connection</a>"
+            }
+          },
+          "warning": {
+            "msg": "Error saving OAuth app settings",
+            "jsondump": "{{0}}"
```

Comment:
A dictionary entry that directly expects an injected value does not make much sense